### PR TITLE
Create knotes2.rb

### PR DESCRIPTION
**Pull request by Chandler-Lu**
After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.
- [x] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).

Additionally, if **adding a new cask**:

- [x] Named the cask according to the [token reference].
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests] for the same cask.
- [x] Checked the cask was not [already refused].
- [x] Checked the cask is submitted to [the correct repo].

[token reference]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/Homebrew/homebrew-cask/pulls
[already refused]: https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues
[the correct repo]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask
[version-checksum]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/sha256.md#updating-the-sha256

### CODE REVIEW

File: Casks/knotes2.rb [hunk 1]
```diff
@@ -0,0 +1,16 @@
+cask 'knotes2' do
+  version '2.1.0'
+  sha256 'd2070cfc96cf168db45e3390866950d1eabb5fdf719bfc85ca558cd68dc151fb'
+
+  url "http://cdn.knotesapp.cn/download/%e7%b3%af%e8%af%8d%e7%ac%94%e8%ae%b0-#{version}.dmg"
+  name '糯词笔记'
```
Comment (vitorgalvao) [inline Casks/knotes2.rb]: It need the english `name` above

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

File: Casks/knotes2.rb (hunk 1 above)
Comment (Chandler-Lu) [reply Casks/knotes2.rb]: Knotes2

Comment (vitorgalvao) [reply Casks/knotes2.rb]: *Above*, not instead of.

**Review by vitorgalvao — DISMISSED** [review]

File: Casks/knotes2.rb [hunk 2]
```diff
@@ -0,0 +1,18 @@
+cask 'knotes2' do
```
Comment (vitorgalvao) [inline Casks/knotes2.rb]: This does not conform to the [token reference](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md). Versions numbers are not to be included. Please follow that document and add the appropriate vendor prefix.

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

File: Casks/knotes2.rb (hunk 1 above)
Comment (Chandler-Lu) [reply Casks/knotes2.rb]: tks~I update it

File: Casks/knotes2.rb [hunk 3]
```diff
@@ -0,0 +1,18 @@
+cask 'knotesapp' do
```
Comment (vitorgalvao) [inline Casks/knotes2.rb]: Please read the [token reference](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md) document, as requested. We don’t add `app` to names of apps, it has to be a **prefix** with the vendor’s name. So if this was made by Apple, for example, you’d call it `apple-knotes`.

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

File: Casks/knotes2.rb (hunk 3 above)
Comment (Chandler-Lu) [reply Casks/knotes2.rb]: Sorry, I change it~

File: Casks/knotes2.rb [hunk 4]
```diff
@@ -0,0 +1,18 @@
+cask 'mhz-knotes' do
```
Comment (vitorgalvao) [inline Casks/knotes2.rb]: Could you explain how you arrived at `mhz`, so we’re clear on the rationale?

Also, don’t forget the file name, it needs to match this token.

Comment (Chandler-Lu) [reply Casks/knotes2.rb]: The developer's company is called "一兆赫兹(Megahertz)"

Comment (vitorgalvao) [reply Casks/knotes2.rb]: Then make it `megahertz`, not `mhz`.

Comment (Chandler-Lu) [reply Casks/knotes2.rb]: OK

**Pull request merged**